javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
+    return this.head === null;
   }
 
+  // Add a Node to the front of a linked list
   insert(value) {
-    //insert a node at the beginning of a given LL
-
-    if (this.isEmpty() === true) {
-      // the list is empty
-      // make a new Node with the value given
-      const node = new Node(value);
-      //   // assign this.head to that new node
-      this.head = node;
-    } else if (this.isEmpty() === false) {
-      // the list is NOT empty
-      // make a new Node with the value given
-      const node = new Node(value);
-      // make the new Node's .next point at the current head
+    const node = new Node(value);
+    if (this.isEmpty()) {
       node.next = this.head;
-      // reassign the head to the new node
-      this.head = node;
     }
+
+    this.head = node;
   }
-  append(value) {
-    // add a value to the end of a linked list
-    const node = new Node(value);
-    // traverse the entire list until I find the one whose next is null
 
+  // Add a Node to the end of a linked list
+  append(value) {
     let currentNode = this.head;
     while (currentNode.next !== null) {
       currentNode = currentNode.next;
     }
-    // when I find that one, make the next my new node
-    currentNode.next = node;
-    // make my new node's next null - already done
+
+    currentNode.next = new Node(value);
   }
-  includes(val) {
-    // Define a method called includes which takes any value as an argument and returns a boolean result depending on whether that value exists as a Node’s value somewhere within the list.
+
+  // Search for a value within the list.
+  includes(value) {
     let currentNode = this.head;
-    // start at the head
     while (currentNode) {
-      // iterate through the whole list until you hit a return statement
-      if (currentNode.value === val) {
+      if (currentNode.value === value) {
         return true;
-        // if the current node contains the value, return true
-      } else if (!currentNode.value === val) {
```

Comment:
Here again, there is no need for the else case. If the `currentNode.value === val` is not true, then it will be false. `if`/`else` is more commonly used for complex comparisons like checking if a string is one of several things.